protocol/x/revshare/types/params.go

```diff
@@ -0,0 +1,32 @@
+package types
+
+import (
+	"github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+)
+
+// DefaultParams returns a default set of `x/revshare` market mapper parameters.
+func DefaultParams() MarketMapperRevenueShareParams {
+	return MarketMapperRevenueShareParams{
+		// TODO: is there a diff address this can be set to?
+		Address:         authtypes.NewModuleAddress(authtypes.FeeCollectorName).String(),
+		RevenueSharePpm: 0,
+		ValidDays:       0,
+	}
+}
+
+// Validate validates `x/revshare` parameters.
+func (p MarketMapperRevenueShareParams) Validate() error {
+	// Address must be a valid address
+	_, err := types.AccAddressFromBech32(p.Address)
+	if err != nil {
+		return ErrInvalidAddress
+	}
+
+	// Revenue share ppm must be less than 1000000 (100%)
+	if p.RevenueSharePpm >= 1000000 {
```

Comment:
nit: Why can't it be 100%?

Reply:
Don't have strong opinion on this since it's never gonna be close to 100% but I was thinking of the implementation to always have something sent to the fee collector